bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/base/Instrument.java

```diff
@@ -258,6 +260,20 @@ public static String getSuffix() {
     public List<TransmissionElement> getComponents() {
         return new ArrayList<>(components);
     }
+
```

Comment:
In particular this part of the Instrument code which represents a series of `TransmissionElement`s (i.e. any optical elements in the light path like filters, gratings etc) is very ITC specific and will stay with the ITC code. However the instruments still contain some code which should be moved to or replaced by code from the main model. This is an ongoing process and I would say I've already reduced the "instrument" code in ITC by about 50%.